widgetssdk/src/main/java/com/glia/widgets/entrywidget/EntryWidgetContract.kt

```diff
@@ -12,16 +14,16 @@ internal interface EntryWidgetContract {
         MESSAGING_LIVE_SUPPORT
     }
 
-    sealed class ItemType(private val order: Int) : Comparable<ItemType> {
-        data object VideoCall : ItemType(0)
-        data object AudioCall : ItemType(1)
-        data object Chat : ItemType(2)
-        data class Messaging(val value: Int) : ItemType(3)
-        data object LoadingState : ItemType(10)
-        data object EmptyState : ItemType(10)
-        data object SdkNotInitializedState : ItemType(10)
-        data object ErrorState : ItemType(10)
-        data object ProvidedBy : ItemType(11)
+    sealed class ItemType(private val order: Int = HIGHEST_ORDER) : Comparable<ItemType> {
+        data object VideoCall : ItemType(order = 0)
+        data object AudioCall : ItemType(order = 1)
+        data object Chat : ItemType(order = 2)
+        data class Messaging(val value: Int) : ItemType(order = 3)
+        data object LoadingState : ItemType()
+        data object EmptyState : ItemType()
+        data object SdkNotInitializedState : ItemType()
+        data object ErrorState : ItemType()
+        data object ProvidedBy : ItemType(order = Int.MAX_VALUE)
```

Comment:
NIT. Maybe it might improve the readability and consistency if you extract this value to a constant as well `private const val LOWEST_ORDER = Int.MAX_VALUE`